checkov/cloudformation/checks/resource/aws/IAMRoleAllowAssumeFromAccount.py

```diff
@@ -23,6 +23,8 @@ def scan_resource_conf(self, conf):
                 assume_role_block = json.loads(assume_role_policy_doc)
             else:
                 assume_role_block = assume_role_policy_doc
+        else:
+            return CheckResult.FAILED
```

Comment:
@Saarett Should be `CheckResult.UNKNOWN` IMO

Reply:
Basically this attribute is a required property, AWS::IAM::Role resources which are missing this attribute are invalid, but I could agree it's unknown in this case